src/legacy/core_plugins/metrics/public/components/aggs/moving_average.js

```diff
@@ -171,72 +199,117 @@ const MovingAverageAggUi = props => {
+                  >
+                    <EuiFieldNumber
+                      step={0.1}
+                      onChange={handleNumberChange('alpha')}
+                      value={model.alpha}
+                    />
+                  </EuiFormRow>
+                </EuiFlexItem>
+              }
+              {
+                (
+                  model.model_type === MODEL_TYPES.WEIGHTED_EXPONENTIAL_DOUBLE ||
+                  model.model_type === MODEL_TYPES.WEIGHTED_EXPONENTIAL_TRIPLE
+                ) && (
+                  <EuiFlexItem>
+                    <EuiFormRow
+                      id={htmlId('beta')}
+                      label={(<FormattedMessage
+                        id="tsvb.movingAverage.beta"
+                        defaultMessage="Beta"
+                      />)}
+                    >
+                      <EuiFieldNumber
+                        step={0.1}
+                        onChange={handleNumberChange('beta')}
+                        value={model.beta}
+                      />
+                    </EuiFormRow>
+                  </EuiFlexItem>
+                )
+              }
+              {
+                model.model_type === MODEL_TYPES.WEIGHTED_EXPONENTIAL_TRIPLE && (
+                  <Fragment>
+                    <EuiFlexItem>
+                      <EuiFormRow
+                        id={htmlId('gamma')}
+                        label={(<FormattedMessage
+                          id="tsvb.movingAverage.gamma"
+                          defaultMessage="Gamma"
+                        />)}
+                      >
+                        <EuiFieldNumber
+                          step={0.1}
+                          onChange={handleNumberChange('gamma')}
+                          value={model.gamma}
+                        />
+                      </EuiFormRow>
+                    </EuiFlexItem>
+                    <EuiFlexItem>
+                      <EuiFormRow
+                        id={htmlId('period')}
+                        label={(<FormattedMessage
+                          id="tsvb.movingAverage.period"
+                          defaultMessage="Period (days)"
+                        />)}
+                      >
+                        <EuiFieldNumber
+                          step={1}
+                          onChange={handleNumberChange('period')}
+                          value={model.period}
+                        />
+                      </EuiFormRow>
+                    </EuiFlexItem>
+                    <EuiFlexItem>
+                      <EuiFormRow
+                        id={htmlId('multiplicative')}
+                        label={(<FormattedMessage
+                          id="tsvb.movingAverage.multiplicative"
+                          defaultMessage="Multiplicative"
+                        />)}
+                      >
+                        <EuiComboBox
+                          placeholder={intl.formatMessage({
```

Comment:
Would be better to replace `intl.formatMessage()` with `<FormattedMessage />` the same as a couple of lines above or (my favourite one) choose using `i18n.translate()`

Reply:
Has been replaced.